src/test/java/com/lpvs/service/QueueServiceTest.java

```diff
@@ -0,0 +1,507 @@
+package com.lpvs.service;
```

Comment:
@o-kopysov We need a copyright statement here. Please take a look at my comment and remedy the PR in the next. 🙏